demos/src/js/overview.js

```diff
@@ -0,0 +1,92 @@
+/*
+Copyright 2015 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.txt
+*/
+
+var demo = demo || {};
+
+(function ($, fluid) {
+
+    "use strict";
+
+    fluid.registerNamespace("demo.chartAuthoring");
+
+    fluid.defaults("demo.chartAuthoring.overview", {
+            gradeNames: ["fluid.viewComponent", "autoInit"],
+            components: {
+                overviewPanel: {
+                    type: "fluid.overviewPanel",
+                    container: "{overview}.container",
+                    options: {
+                        modelListeners: {
+                            "": {
+                                listener: "{cookieStore}.set",
+                                args: ["{change}.value"]
+                            }
+                        },
+                        model: {
+                            showPanel: {
+                                expander: {
+                                    funcName: "fluid.get",
```

Comment:
This is a bit unnecessarily complicated. I suggest instead you just set up a "cookieStoreModel" (or more generically, just a "storeModel") component which automatically syncs its entire model back and forth to the store, and then operate both of these links by model relay. At the same time you can then take care of the asymmetry here - that is, that it writes the entire model but only reads one flag.